### ISSUE TITLE
Enhancement - Termination Reason

### ISSUE THREAD

**Issue by Carolane-IA** [ApexaCorp/feedback-portal#76]
It would be usefull if a terminaison reason named Block of business sold (Vente de clientèle in French) because when an advisor sell all his block of business we often terminate his contract because he will also get retired.

**Comment by Bobbisue-HUB** [comment]
I would agree - this is a valid reason for inactivation (termination) as far as HUB is concerned.

**Comment by Angela-MLI** [comment]
Manulife does not terminate an advisor's code when their block of business is sold.  We have experienced where the advisor will be continuing to sell, but is moving or has some other business reason for the sale
